threads/README.md

```diff
@@ -0,0 +1,21 @@
+# A pure javascript shim for WASI Preview 1 threads
+
```

Comment:
```suggestion

> [!WARNING]
> The code in this directory is less production ready than the main browser_wasi_shim code.

```

Reply:
thanks!

Reply:
I'm sorry. I forgot to mention that cross-origin isolation is required. Could you please add that?